week6-rnn-tensorflow/01b_rnn_flask_js/server.py

```diff
@@ -27,23 +27,28 @@ def sample(preds, temperature=1.0):
     probas = np.random.multinomial(1, preds, 1)
     return np.argmax(probas)
 
-# Load in the model
-model = load_model('model.h5')
+# Load the models
+model1 = load_model('model.h5')
+model2 = load_model('metamorphosis.h5')
```

Comment:
I love this idea and elegance of switching between the two in one app. I think it might be nice for this to be a second example so that this example can focus on the core implementation of RNN generation in python with seed text from p5 and the results rendered in p5.